### ISSUE TITLE
Postman project: extra characters at end of codes

### ISSUE THREAD

**Issue by uy-rrodriguez** [IHTSDO/snowstorm#253]
Dear all,

When trying out the sample Postman project (https://github.com/IHTSDO/snowstorm/blob/master/docs/using-the-fhir-api.md), some queries fail because of an extra "Enter" character at the end of the codes. For e.g., the request "CodeSystem> Get a concept based on identifier" fails with
>Failed to call access method: org.snomed.snowstorm.fhir.services.FHIROperationException: Only numeric SNOMED CT identifiers are currently supported

This is due to extra "%0A" characters at the end. I feel this is caused by the way Postman requests have been created, because every item seems to have this character as part of the last parameter.

The windows Postman application shows a "line break" icon at the end of the value. You can also see it in the live documentation https://documenter.getpostman.com/view/462462/S1TVXJ3k. For e.g., the same lookup "Get a concept based on identifier" translates to the URl below, according to what Postman prints on the browser:
> curl --location --request GET 'https://snowstorm-fhir.snomedtools.org/fhir/CodeSystem/$lookup?system=http://snomed.info/sct&code=427623005%0A'

**Comment by pgwilliams** [comment]
Yes great, thanks for spotting that @uy-rrodriguez !  I've fixed the postman collection and updated the links to it in the documentation.  See https://github.com/IHTSDO/snowstorm/blob/develop/docs/using-the-fhir-api.md    As per our policy, this issue will remain open until we've published the next release.

**Comment by kaicode** [comment]
@pgwilliams feel free to correct documentation directly in master. Only code changes need to go through develop and be tied to a release.
